lib/src/components/NxToast/__tests__/NxToastContainer.test.tsx

```diff
@@ -0,0 +1,161 @@
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { useState } from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { screen, render, fireEvent } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
+
+import {reject, propEq } from 'ramda';
+
+import useToggle from '../../../util/useToggle';
+
+import NxToastContainer from '../NxToastContainer';
+import { NxToastContainerProps } from '../types';
+import NxToast from '../NxToast';
+import { NxErrorAlert } from '../../NxAlert/NxAlert';
+
+describe('NxToastContainer', function() {
+  const simpleProps: NxToastContainerProps = {
+    children: []
+  };
+
+  const quickRender = rtlRender<NxToastContainerProps>(NxToastContainer, simpleProps),
+      renderEl = rtlRenderElement(NxToastContainer, simpleProps);
+
+  it('renders a <div> with a class of nx-toast__wrapper', function() {
+    const component = quickRender();
+    const toastWrapper = component.container.children[0];
+
+    expect(toastWrapper.nodeName).toEqual('DIV');
+    expect(toastWrapper).toHaveClass('nx-toast__wrapper');
+  });
+
+  it('contains a<div> with a class of nx-toast__container', function() {
+    const component = renderEl();
+    const toastContainer = component?.querySelector('.nx-toast__container');
+
+    expect(toastContainer).toBeInTheDocument();
+    expect(component).toContainElement(toastContainer as HTMLElement);
+  });
+
+  it('tests NxToast mounting inside NxToastContainer, and unmounting when close button is clicked',
+      async function() {
+
+        const user = userEvent.setup();
+
+        const MountingBehavior = () => {
+          const [showToast, toggleToast] = useToggle(false);
+          return (
+            <>
+              <button type="button" onClick={()=>toggleToast()}>Show Toast</button>
+              <NxToastContainer>
+                {showToast &&
+                <NxToast onClose={()=>toggleToast()}>
+                  <NxErrorAlert>message</NxErrorAlert>
+                </NxToast>
+                }
+              </NxToastContainer>
+            </>
+          );
+        };
+        const { container } = render(<MountingBehavior/>);
+
+        expect(screen.queryByRole('alert')).not.toBeInTheDocument();
+
+        const button = screen.getByRole('button', {name: 'Show Toast'});
+        await user.click(button);
+
+        expect(screen.getByRole('alert')).toBeVisible();
+
+        const toast = container.querySelector('.nx-toast');
+        const closeBtn = screen.getByRole('button', {name: 'Close'});
+        await user.click(closeBtn);
+        fireEvent.animationEnd(toast as Element);
+
+        expect(toast).not.toBeInTheDocument();
+      });
+
+  it('tests focusing when NxToasts mount and unmount', async function() {
```

Comment:
Test name needs to be improved.

Reply:
> Test name needs to be improved.

All tests inside NxToastContainer have been renamed here [124d31a](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/124d31a171f3c37e46b17f1a6acb9e74d964e45d)